src/utils.ts

```diff
@@ -63,16 +63,18 @@ const TX_CACHE_SIZE: number = parseInt(process.env.TX_CACHE_SIZE as string) || 2
  * @param txIds - List of transactions to download
  * @param data - Downloaded transactions, used while being called recursively
  */
-export const recursivelyDownloadTx = async (blockId: string, txIds: string[] = [], data: FullTx[] = []): Promise<FullTx[]> => {
+export const recursivelyDownloadTx = async (blockId: string, txIds: string[] = [], data = new Map<string, FullTx>()): Promise<Map<string, FullTx>> => {
   if (txIds.length === 0) {
     return data;
   }
 
   const txId: string = txIds.pop() as string;
-  const network = process.env.NETWORK || 'mainnet';
+  const network: string = process.env.NETWORK || 'mainnet';
+
+  if (IGNORE_TXS.has(network)) {
```

Comment:
Should we allow an unknown network? And maybe the txs on `IGNORE_TXS` should be in the wallet-lib constants.

Reply:
I don't think we should allow an unknown network. What do you think, @pedroferreira1 ?

I agree that the txs should be in the wallet-lib, I will create an issue there.

Reply:
In the lib we already raise exception if we try to use a network different than 'mainnet' or 'testnet', so I think it's fine to handle only them.

About the txs in the lib I agree and I believe we've created an issue about that (or should've created) because we've already discussed this